### DISCUSSION TITLE
Fix for OOTP 25 in the works.

### DISCUSSION THREAD

**Discussion by sleffew80** [sleffew80/OOTPDBTools#2] [Announcements]
Currently, OOTPDB Tool crashes when converting a database from OOTP 25. I should have a fix relatively soon.

**Comment by sleffew80** [comment]
Fixed crash when trying to open OOTP 25 database files.

**Comment by mwbsix** [comment]
Hello, I can't seem to get the converter app to launch on Mac using the control-click "open" option, I get an error "The application “OOTP Database Converter” can’t be opened."

**Reply by sleffew80** [reply]
I've updated the download link and it should work now. Just make sure you Ctrl+Click the app the first time you open it as I am not a verified apple developer.

**Reply by mwbsix** [reply]
Thanks very much for creating this utility and for applying the update!

**Comment by plannine2000** [comment]
Are you still around. I have been loving the tool with OOTP25 but it looks like OOTP changed the DB again for OOTP26.

**Reply by sleffew80** [reply]
Database Converter has been updated for OOTP 26. Both the Windows and macOS versions can be downloaded under releases.
